src/styles/main.scss

```diff
@@ -0,0 +1,27 @@
+.main {
+  --column-count: 1;
+
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
```

Comment:
you need to use your var here
<img width="1255" alt="image" src="https://github.com/user-attachments/assets/fa5e1174-5ba5-4f4e-809f-ebc68b4cd399">

instead of auto-fill